src/commands.cpp

```diff
@@ -3925,6 +3925,19 @@ void CommandGetUserQuota::procresult()
                         ns->version_bytes = client->json.getint();
                         ns->version_files = client->json.getint();
 
+#ifdef _DEBUG
```

Comment:
Please, check if you really want to keep this block in place for a while. I have no problem with that, despite we should eventually remove it.

Reply:
Then we should replace the `_DEBUG` by a more standard macro, such as `NDEBUG` (defined by C standard) or simple `DEBUG` (defined by our projects). That way, not only Windows developers will run the code, but any debug build (ie. SDK/MEGAchat devs using Linux, Android and iOS devs)